package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "pm2 start .pm2.json",
     "start:debug": "SILEX_DEBUG=true node dist/server/server.js",
-    "build": "npm run build:css && npm run build:html && npm run build:js:release ",
```

Comment:
yeah, so strange..., i can't remember exactly how its built, but anyway do you have any plan to migrate to babel?